File: ApiBancoCelular/APIBancoPessoa/src/model/Banco.js
const { Client } = require('pg');
class Banco {
  static conexao;
  static init() {
    try {
      this.conexao = new Client({
        host: '127.0.0.1',
        port: 5432,
        database: 'WEB2',
        user: 'postgres',
        password: 'ifsp',
      });
      this.conexao.connect();
    }
    catch (erro) {
      console.log("Erro de conexao : " + erro);
    }
  }
}
module.exports = Banco;

File: CRUD/src/model/Banco.py
# pip install "psycopg[binary]"
import psycopg

class Banco:
    def __init__(self):
        try:
            self.__conexao =  psycopg.connect(
                        host="127.0.0.1",
                        dbname="LPB",
                        user="postgres",
                        password="ifsp",
                        port=5432
                        )
            self.__cursor = self.__conexao.cursor()
        except Exception as erro:
            raise Exception(" Erro na conexao com o banco "+ str(erro))

    @property
    def conexao(self):
        return self.__conexao

    @property
    def cursor(self):
        return self.__cursor
  